package.json

```diff
@@ -109,7 +109,7 @@
     "@mean-finance/dca-v2-core": "^3.0.0",
     "@mean-finance/dca-v2-periphery": "^3.0.0",
     "@mean-finance/oracles": "^2.0.0",
-    "@mean-finance/sdk": "0.0.22",
+    "@mean-finance/sdk": "^0.0.26",
```

Comment:
I would like to keep this hardfixed since patch versions on `0.0.x` in standard versioning can mean breaking changes